lexer/lexer.go

```diff
@@ -207,6 +208,12 @@ func (l *Lexer) eatCurrentLine() {
 }
 
 func (l *Lexer) eatMultiLineComment() {
+	for {
+		l.eatUntil('*')
+		if char, _ := l.NextChar(); char == '/' {
+			break
```

Comment:
I added a panic in eatUntil so it should eventually panic once its exhausted all asterisks after not finding an asterisk with a / following it